test/snapshot/__snapshots__/graceful.test.js.snap

```diff
@@ -300,19 +300,19 @@ Program {
           "kind": "propertystatement",
           "properties": [
             Property {
-              "attrGroups": null,
+              "attrGroups": [],
               "kind": "property",
               "name": Identifier {
                 "kind": "identifier",
                 "name": "onst",
               },
-              "nullable": Name {
```

Comment:
The snapshot show that what was set was incorrect: instead of a boolean value we had some objects.